src/utils/topics/removePrefixTopic.js

```diff
@@ -1,15 +1,15 @@
-import { PREFIX_TOPIC } from "../constants"
-import { convertString } from "../string/StringUtils"
+import {PREFIX_TOPIC} from '../constants';
+import {convertString} from '../string/StringUtils';
 
-const removePrefixTopic = topicWithPrefix => {
-    if(topicWithPrefix === null || topicWithPrefix === undefined) return
+const removePrefixTopic = (topicWithPrefix) => {
+  if (topicWithPrefix === null || topicWithPrefix === undefined) return;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `removePrefixTopic` returns `undefined` when the input `topicWithPrefix` is `null` or `undefined`. This could potentially lead to unexpected behavior in the calling code if it doesn't handle `undefined` values properly. It would be better to return a default value, like an empty string, to ensure consistent return types.
```diff
-  if (topicWithPrefix === null || topicWithPrefix === undefined) return;
+  if (topicWithPrefix === null || topicWithPrefix === undefined) return '';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->